zmscitizenapi/public/doc/partials/definitions.yaml

```
@@ -0,0 +1,33 @@
+  officesAndServices:
+    $ref: "schema/citizenapi/officesAndServices.json"
+  officesByService:
+    $ref: "schema/citizenapi/officesByService.json"
+  scope:
+    $ref: "schema/citizenapi/scope.json"
+  scopes:
+    $ref: "schema/citizenapi/scopes.json"
+  services:
+    $ref: "schema/citizenapi/serviceList.json"
+  servicesByOffice:
+    $ref: "schema/citizenapi/servicesByOffice.json"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)